src/settings/App.tsx

```diff
@@ -54,6 +64,12 @@ export class App extends Component<{}, AppState> implements OC.Plugin<OC.Search.
 		this.api.listGroups().then((groups) => {
 			this.setState({groups});
 		});
+		this.api.isAdminNextcloud().then((isAdminNextcloud) => {
+			this.setState({isAdminNextcloud});
+		});
+		this.api.checkAppsBasedOnGroupfolders().then((checkAppsInstalled) => {
+			this.setState({checkAppsInstalled})
+		});
```

Comment:
Use initial state https://github.com/nextcloud/nextcloud-initial-state

Reply:
I think I resolved this issue. Please, look at my comment here : https://github.com/nextcloud/groupfolders/pull/2072#discussion_r982380393